spec/support/reporters/test-coverage.mjs

```diff
@@ -0,0 +1,24 @@
+import fsPromises from 'node:fs/promises';
+import {URL} from 'url';
+import path from 'node:path';
+
+export default class TestCoverageReporter {
+  suiteDone({properties}) {
```

Comment:
Could we add a code comment here, something along the lines of:
`Runs after every top level Jasmine describe-block`

Reply:
Done. I checked and it runs after every Jasmine block, not only top-level ones.